graylog2-web-interface/src/components/common/Spoiler.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { Spoiler as MantineSpoiler } from '@mantine/core';
+
+type Props = React.PropsWithChildren<{
+  maxHeight: number,
+  className?: string,
```

Comment:
Do we really want to expose the `className` prop? I think in general we encourage using styled components for styling changes to a component instead?

Reply:
I added the `className` prop to be able to style the component via `styled(Spoiler)`. In the case where I currently need this component I had to adjust the `word-break` behaviour.

Reply:
You are right. I forgot that `styled-components` uses it to pass the class it had generated. :facepalm: